### ISSUE TITLE
Nested Object error: multiple types named

### ISSUE THREAD

**Issue by miyagi389** [incetarik/nestjs-graphql-zod#5]
I tried the [Nested Object example](https://github.com/incetarik/nestjs-graphql-zod#nested-object).
However, I get the following error

```
Error: Schema must contain uniquely named types but contains multiple types named "ExampleUser".
```

I have created a sample source, could you please check it?
https://github.com/miyagi389/nestjs-graphql-zod-examples/blob/main/src/app.resolver.ts

Here are the results of the sample run.

```bash
$ git clone https://github.com/miyagi389/nestjs-graphql-zod-examples
$ cd nestjs-graphql-zod-examples
$ yarn
$ yarn start
yarn run v1.22.19
$ nest start
[Nest] 84291  - 09/27/2022, 5:52:50 PM     LOG [NestFactory] Starting Nest application...
[Nest] 84291  - 09/27/2022, 5:52:50 PM     LOG [InstanceLoader] AppModule dependencies initialized +18ms
[Nest] 84291  - 09/27/2022, 5:52:50 PM     LOG [InstanceLoader] GraphQLSchemaBuilderModule dependencies initialized +0ms
[Nest] 84291  - 09/27/2022, 5:52:50 PM     LOG [InstanceLoader] GraphQLModule dependencies initialized +0ms
[Nest] 84291  - 09/27/2022, 5:52:50 PM     LOG [RoutesResolver] AppController {/}: +2ms
[Nest] 84291  - 09/27/2022, 5:52:50 PM     LOG [RouterExplorer] Mapped {/, GET} route +1ms

/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/graphql/type/schema.js:219
        throw new Error(
              ^
Error: Schema must contain uniquely named types but contains multiple types named "ExampleUser".
    at new GraphQLSchema (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/graphql/type/schema.js:219:15)
    at GraphQLSchemaFactory.create (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/graphql/dist/schema-builder/graphql-schema.factory.js:40:24)
    at GraphQLSchemaBuilder.generateSchema (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/graphql/dist/graphql-schema.builder.js:35:52)
    at GraphQLSchemaBuilder.build (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/graphql/dist/graphql-schema.builder.js:22:31)
    at GraphQLFactory.mergeWithSchema (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/graphql/dist/graphql.factory.js:29:69)
    at ApolloDriver.start (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/apollo/dist/drivers/apollo.driver.js:19:51)
    at GraphQLModule.onModuleInit (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/graphql/dist/graphql.module.js:105:36)
    at callModuleInitHook (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/core/hooks/on-module-init.hook.js:51:9)
    at NestApplication.callInitHook (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/core/nest-application-context.js:178:13)
    at NestApplication.init (/Users/miyagi389/WebStormProjects/nestjs-graphql-zod-examples/node_modules/@nestjs/core/nest-application.js:96:9)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by incetarik** [comment]
Thank you for your report @miyagi389. Checking it now and will push an immediate fix if there is an error.

**Comment by incetarik** [comment]
@miyagi389 Should be fixed now in `2.0.1`, please update your version and try again

**Comment by miyagi389** [comment]
I have confirmed that it has been fixed.
Thank you for your response!
